CMakeLists.txt

```diff
@@ -137,9 +137,14 @@ ENDIF()
 # Check for libtiff
 ################################################################################
 
-find_package(TIFF REQUIRED)
-if(NOT TIFF_FOUND)
-  message(SEND_ERROR "libtiff dependency not found!")
+option(DISABLE_TIFF_IS_STRONGLY_DISCOURAGED "Disable TIFF support (strongly discouraged !)" OFF)
```

Comment:
Is this the actual flag for disabling tiff support? So you would run cmake with `cmake -DDISABLE_TIFF_IS_STRONGLY_DISCOURAGED` or something like that?

Reply:
> Is this the actual flag for disabling tiff support?

yes, this is intended to be cumbersome so that only people realize they probably do something they shouldn't. You need to specify -DDISABLE_TIFF_IS_STRONGLY_DISCOURAGED=ON
I'd be open to use a more normal ENABLE_TIFF=ON approach if you think we're going too far

Reply:
I think it is fine to have TIFF enabled by default. I would go for a more neutral wording in the off-switch. "disable tiff is strongly discouraged" doesn't really sound like an on/off switch to me.  `DISABLE_TIFF` would be more suitable in my opinion. I would however encourage plastering the screen with warnings when that switch is turned on.

Reply:
done